### PR TITLE
Fix custom blocks not using custom classes

### LINKED ISSUE #6366: Custom blocks do not add class names defined in blocks
## Description

When you create a custom block the class name added is just `custom-block` instead of the class name defined in the config.

### Steps to reproduce

Clone [this demo repo](https://github.com/giraffesyo/gatsby-bug-with-custom-blocks) and run `gatsby develop`. I put a gatsby custom block at the bottom of the hello world post (`http://localhost:8000/hello-world/`)

### Expected result

The string provided as a class name should be used instead of `custom-block`

### Actual result

Only `custom-block` is added

### Environment

(originally ran into this in a windows environment, but made the demo on my mac)
```

  System:
    OS: macOS 10.14
    CPU: x64 Intel(R) Core(TM) i7-7920HQ CPU @ 3.10GHz
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 8.11.3 - ~/n/bin/node
    Yarn: 1.7.0 - ~/n/bin/yarn
    npm: 6.1.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 67.0.3396.99
    Safari: 12.0
  npmPackages:
    gatsby: next => 2.0.0-beta.20 
    gatsby-plugin-feed: next => 2.0.0-beta.3 
    gatsby-plugin-google-analytics: next => 2.0.0-beta.2 
    gatsby-plugin-offline: next => 2.0.0-beta.3 
    gatsby-plugin-react-helmet: next => 3.0.0-beta.3 
    gatsby-plugin-sharp: next => 2.0.0-beta.2 
    gatsby-plugin-typography: next => 2.2.0-beta.2 
    gatsby-remark-copy-linked-files: next => 2.0.0-beta.2 
    gatsby-remark-custom-blocks: ^2.0.0-beta.3 => 2.0.0-beta.3 
    gatsby-remark-images: next => 2.0.1-beta.3 
    gatsby-remark-prismjs: next => 3.0.0-beta.3 
    gatsby-remark-responsive-iframe: next => 2.0.0-beta.2 
    gatsby-remark-smartypants: next => 2.0.0-beta.2 
    gatsby-source-filesystem: next => 2.0.1-beta.3 
    gatsby-transformer-remark: ^2.1.1-beta.2 => 2.1.1-beta.2 
    gatsby-transformer-sharp: next => 2.1.1-beta.3 

```

### PR DESCRIPTION
**Pull request by giraffesyo**
Fixes #6366

### CODE REVIEW

**Comment by m-allanson** [comment]
This looks good, thanks @giraffesyo. 

What do you think about changing the config format of `gatsby-remark-custom-blocks` to match the format expected by `remark-custom-blocks`? Then we can just pass the options straight through, and it won't be a breaking change if this feature wasn't working before.

**Comment by giraffesyo** [comment]
@m-allanson  If you pass the options as documented on the remark-custom-blocks it works as expected, as I noted in #6366. So, in that case it would just be a documentation change. My question is, was there originally a reason to have a different format from `remark-custom-blocks`?

**Comment by m-allanson** [comment]
:+1: Yeah let's do this as a documentation change. 

> My question is, was there originally a reason to have a different format from remark-custom-blocks?

I don't know for sure, but I can't think of a good reason why having a different format would be useful here.

File: packages/gatsby-remark-custom-blocks/README.md [hunk 1]
```diff
@@ -2,14 +2,27 @@
 
 Adds custom blocks to `MarkdownRemark` using [remark-custom-blocks](https://github.com/zestedesavoir/zmarkdown/tree/master/packages/remark-custom-blocks).
 
+Unlike
```
Comment (m-allanson) [inline packages/gatsby-remark-custom-blocks/README.md]: delete?

Comment (giraffesyo) [reply packages/gatsby-remark-custom-blocks/README.md]: Whoops, had a train of thought there that went astray. I added a commit to fix this. Let me know if there should be any changes.

**Comment by giraffesyo** [comment]
Thanks @m-allanson!

**Pull request merged**